helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -557,9 +557,15 @@ starrocksCnSpec:
     # You must set name when you set storageClassName
     # Note: Because hostPath field is not supported here, hostPath is not allowed to be set in storageClassName.
     storageClassName: ""
-    # the storage size of persistent volume for data.
+    # the storage size of per persistent volume for data.
     storageSize: 1Ti
     # If storageMountPath is empty, the storageMountPath will be set to /opt/starrocks/cn/storage.
+    # the number of persistent volumes for data.
+    # if its value is greater than 1:
```

Comment:
need to revise the comments together with the storageMountPath
* storageCount == 1, what the `storageMountPath` means and what the `storage_root_path` should be configured
* storageCount > 1, what the `storageMountPath` means and what the `storage_root_path` should be configured.